src/config.rs

```diff
@@ -240,7 +240,7 @@ pub struct EnclaveConfig {
     #[serde(default = "default_true")]
     pub trx_logging: bool,
     #[serde(default)]
```

Comment:
Should probably default to true now 
```suggestion
    #[serde(default = "default_true")]
```

Reply:
Ahh thanks good catch